index.html

```diff
@@ -240,5 +240,34 @@ <h3>Skills</h3>
         </li>
       </ul>
     </section>
+
+    <section class="form-container">
+         <div class="form-text">
+           <h1>Contact me</h1>
```

Comment:
- [ ] I think it would be better here if you use `h2` or `h3` to follow the SEO optimize in the semantic of HTML and it would be better here if you stick with the same font weight and color so kindly Fix it to be the same with the Figma design

- [ ] As I can see here also the background image is not aligned the same as the Figma design please add it to match the Figma 


- Your design
![image](https://user-images.githubusercontent.com/58553711/179062629-151d46ab-f8b4-4edc-bd40-d2979538f8f0.png)

- Figma design
![image](https://user-images.githubusercontent.com/58553711/179062741-69d616bb-af7f-42ef-b679-cd95bad8c70d.png)